presto-docs/src/main/sphinx/develop/functions.rst

```diff
@@ -321,3 +321,66 @@ function follows:
   is used when performing a ``GROUP BY`` aggregation, and an implementation
   will be automatically generated for you, if you don't specify a
   ``AccumulatorStateFactory``
+
+
+Advanced Use Cases
+------------------
+
+Raw Block Inputs
+^^^^^^^^^^^^^^^^
+
+Both scalar and aggregation function annotations allow you to define methods
+which operate on native types. In Java, these native types are ``boolean``,
+``Slice``, and ``long``. However, for parameterized implementations or
+parametric types, the standard Java types can't always be used as they aren't
```

Comment:
```suggestion
parametric types, the standard Java types can't always be used as they can't
```

Reply:
Do you think it sounds a little weird reading out the sentence having "can't" repeated twice? 

With all of your suggested changes the sentence would be:

> For parameterized implementations or parametric types, the standard Java types can't always be used as they can't fully represent the input data.

I think keeping the "aren't"s would make it sound better IMO?

> For parameterized implementations or parametric types, the standard Java types can't always be used as they aren't able to represent the input data.

I don't have a strong preference, but usually when writing I try to avoid to repeating the same word within the same sentence

Reply:
Good point! I suggested changing the "aren't" to "can't" to move away from the passive "aren't able to", and for shorter. That said, this one is very much a suggestion! 

I had a couple of new suggestions in a new review, so if you'd like to change this back to "aren't able to" in what I think is the final go-round, please do so!